chapter_1/Guardfile

```diff
@@ -0,0 +1,21 @@
+# A sample Guardfile
```

Comment:
This file should not exist.  We only have one Guardfile, which is in the parent directory (lrthw_exercises).